widget/embedded/src/hooks/useSwapInput.ts

```diff
@@ -184,9 +184,7 @@ export function useSwapInput({
     }
     if (shouldSkipRequest) {
       setLoading(false);
-      if (selectedQuote) {
-        resetQuote();
-      }
+      resetQuote();
```

Comment:
Can you explain this change?

Reply:
I fixed it. The route might not have been chosen, but there might be a warning or error.